course-mw/course-actors-common/src/main/java/org/sunbird/learner/util/Util.java

```diff
@@ -47,9 +47,9 @@ private Util() {}
   /** This method will initialize the cassandra data base property */
   private static void initializeDBProperty() {
     dbInfoMap.put(
-        JsonKey.LEARNER_COURSE_DB, getDbInfoObject(COURSE_KEY_SPACE_NAME, "user_enrolments"));
+            JsonKey.LEARNER_COURSE_DB, getDbInfoObject(COURSE_KEY_SPACE_NAME, "user_enrolments_v2"));
```

Comment:
Do not replace... 
Let's create new ExtendedUtil.java file and have the properties there...